isis/appdata/export/TGOCaSSIS.tpl

```diff
@@ -244,39 +244,27 @@
   <File>
    <file_name>{{MainLabel.IsisCube.Archive.FileName.Value}}.img</file_name>
   </File>
-  <Array_3D_Image> {# TODO - Can this be 2D if there are no Bands ?? #}
-   <local_identifier>Array_3D_Image</local_identifier>
-   <offset unit="byte">0</offset>
-   <axes>3</axes>
-   <axis_index_order>Last Index Fastest</axis_index_order>
-   <Element_Array>
-    <data_type>{{pixelType}}</data_type>
-    <scaling_factor>1.0</scaling_factor>
-    <value_offset>0.0</value_offset>
-   </Element_Array>
-   <Axis_Array>
-    <axis_name>Band</axis_name>
-    <elements>{{MainLabel.IsisCube.Core.Dimensions.Bands.Value}}</elements>
-    <sequence_number>1</sequence_number>
-   </Axis_Array>
-   <Axis_Array>
-    <axis_name>Line</axis_name>
-    <elements>{{MainLabel.IsisCube.Core.Dimensions.Lines.Value}}</elements>
-    <sequence_number>2</sequence_number>
-   </Axis_Array>
-   <Axis_Array>
-    <axis_name>Sample</axis_name>
-    <elements>{{MainLabel.IsisCube.Core.Dimensions.Samples.Value}}</elements>
-    <sequence_number>3</sequence_number>
-   </Axis_Array>
-   <Special_Constants>
```

Comment:
I *think* these should still be in the output. Why did they need to be removed? (Did it break the import?)

Reply:
It is probably a good Idea to comment this out as well. I think I realized I could comment things after removing this.

Reply:
Added back in Array_3D_Image but left the section in comments. There should be an if statement added at some point as to whether to add Array_2D or Array_3D.